sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -206,15 +210,114 @@ def convert_genbank_to_sbol3(
             doc.write(fpath=sbol3_file, file_format=sbol3.SORTED_NTRIPLES)
         return doc
 
+    def convert_sbol3_to_genbank(
+        self,
```

Comment:
Minor style point: this is not the usual indentation for long argument lists.

Reply:
I have edited this in the latest commit, but am not sure if even it is the correct way.
Maybe we should decide on a py formatter to handle these style issues?